recipes/libpq/all/conanfile.py

```diff
@@ -79,9 +77,25 @@ def _configure_cmake(self):
         return cmake
 
     def build(self):
-        if self.settings.os == "Windows" and self.settings.compiler == "Visual Studio":
-            cmake = self._configure_cmake()
-            cmake.build()
+        if self.settings.compiler == "Visual Studio":
+            # https://www.postgresql.org/docs/8.3/install-win32-libpq.html
+            # https://github.com/postgres/postgres/blob/master/src/tools/msvc/README
+            if not self.options.shared:
+                tools.replace_in_file(os.path.join(self._source_subfolder, "src", "tools", "msvc", "MKvcbuild.pm"),
```

Comment:
What about zlib, openssl support? Doesn't it need any changes regards to these depenenceies? 

I see config_default.pl (https://github.com/postgres/postgres/blob/c114229ca2519620703a4be4e38181290cad8c0a/src/tools/msvc/config_default.pl#L23) has undefined openssl path by default.

Reply:
good point, I'll take a look to implement these options

Reply:
added openssl and zlib handling
also, there are other deps, but as they aren't handled for all platforms, we may introduce them eventually, on demand